tests/run.sh

```diff
@@ -174,6 +174,9 @@ if [ -n "$DO_COVERAGE" ] && [ "$DO_COVERAGE" != 0 ]; then
     sed "1 s~^~require('luacov.runner')('$source_dir/.luacov'); \0~" \
         "$RC_FILE" > "$tmp_files/awesomerc.lua"
     RC_FILE=$tmp_files/awesomerc.lua
+
+    # Use xresources theme for more coverage.
+    sed -i 's~default/theme~xresources/theme~' "$RC_FILE"
```

Comment:
i am more curious, if such all-theme-iteration test can be easily integrated to the current testsuite so its execution will be traced by coverage measurement tool


so mb instead of sed-ing just do everything on lua side:
for theme in all-themes
1) flush beautiful theme table
2)  beautiful.init(theme)
nothing raised exceptions -- we are good
3) awful.util.restart()

or awesome can't be reloaded during testsuite execution? if so i still think what just flushing beautiful theme table then could be enough